cluster-autoscaler/cloudprovider/azure/azure_manager.go

```diff
@@ -26,13 +26,18 @@ import (
 	"time"
 
 	"github.com/Azure/go-autorest/autorest/azure"
+	"k8s.io/apimachinery/pkg/util/wait"
```

Comment:
Not requiring this in master due to https://github.com/kubernetes/autoscaler/pull/6550